config/initializers/config.rb

```diff
@@ -38,5 +38,6 @@
   'APPOINTMENT_NOTIFICATION_FACILITY_IDS',
   'ENABLE_SMS_REMINDERS',
   'ANALYTICS_DASHBOARD_CACHE_TTL',
-  'IHCI_ORGANIZATION_UUID'
+  'IHCI_ORGANIZATION_UUID',
+  'ENABLE_SYNC_ENCOUNTERS'
```

Comment:
It's not necessary for this key to be required. 

Feature toggles are assumed to be off unless they are explicitly set to `true`